root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4529,14 +4529,18 @@ const testData = [
   {
                      input_url: 'https://vimeo.com/ondemand/inconcert/193518106?autoplay=1',
             expected_clean_url: 'https://vimeo.com/ondemand/inconcert',
+             input_entity_type: 'recording',
        input_relationship_type: 'downloadpurchase',
        only_valid_entity_types: ['recording', 'release'],
+ expected_relationship_options: ['downloadpurchase', 'streamingpaid'],
   },
   {
                      input_url: 'https://vimeo.com/ondemand/inconcert#comments',
             expected_clean_url: 'https://vimeo.com/ondemand/inconcert',
+             input_entity_type: 'recording',
        input_relationship_type: 'streamingpaid',
        only_valid_entity_types: ['recording', 'release'],
+ expected_relationship_options: ['downloadpurchase', 'streamingpaid'],
   },
```

Comment:
These examples are wrong. The given URL `https://vimeo.com/ondemand/inconcert` should be linked using both `downloadpurchase`and `streamingpaid` relationship types. I don’t know if that applies to all Vimeo pages though. Sure things are:

* We need to know if all of Vimeo On Demand pages provide always both options or just any combination of these.
* Relationship types auto-selection for Vimeo On Demand URLs must be fixed to allow both relationship types at the same type (or to enforce selecting both if that matches Vimeo On Demand’s offer).
* Options example must be updated accordingly.

Reply:
It has to support all three options, as per the old discussion on https://tickets.metabrainz.org/browse/MBS-11952 - will fix accordingly.